signup.html

```diff
@@ -3,63 +3,51 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      as="style"
-      crossorigin
-      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
-    />
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
     <link rel="stylesheet" href="/src/signup.css" />
     <title>Linkbrary</title>
   </head>
   <body>
     <main>
       <div class="login-form">
         <div class="header">
-          <a href="/"
-            ><img
-              class="header-logo"
-              src="/public/img/logo.svg"
-              alt="홈 연결 로고"
-          /></a>
+          <a href="/"><img class="header-logo" src="/public/img/logo.svg" alt="홈 연결 로고" /></a>
           <p class="header-message">
             이미 회원이신가요?
             <a class="header-signup-link" href="signin.html">로그인 하기</a>
           </p>
         </div>
-        <form action="">
+        <form id="login-form" action="">
```

Comment:
form태그에서 사용하지 않는 action 속성은 제거해주는게 좋을 것 같아요!
사용하지 않는 데 남아있으면 다른 사람이 코드를 볼 때 혼란스러울 수 있어요~
```suggestion
        <form id="login-form">
```